numjuggler/parser.py

```diff
@@ -1150,44 +1155,54 @@ def is_blankline(l):
     """
     return l.strip() == ''
 
+if six.PY2:
```

Comment:
Why the attempt to work with the dump file is wrapped with the ``if six.PY2 ...`` clause? Does it fails under Python3 or has no influence to the performance? (Under Python2 the use of dump file gives considerable gain in reading time, as tested on a ~ 40 MB input file)

Reply:
cPickle is not available in Python3, only the pickle module.Trying just to switch to pickle, I've met some issues with byte/unicode conversions and didn't have enough time to solve this. There's no performance issues for me, even with ITER models.
For caching I've used joblib. But these caches may be mysterious if not all dependencies are accounted on a cache update. I usually drop the caches every second run, when debugging the code.